### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by Htcicv** [VAST-AI-Research/TriplaneGaussian#22]
Dear author, I admire your work so much!!!

Could you please release your training code? I want to make my own dataset to train a custom model.

Thanks again for your excellent work.

**Comment by Mike001-wq** [comment]
Me too, I want to cite on the work and use a custom model.

**Comment by PavanMantripragada** [comment]
Same. Training code would help others replicating your work for comparisons on custom dataset

**Comment by hiyyg** [comment]
Same. If the training code is provided, we will cite the paper.

**Comment by Cindy0725** [comment]
Same, hope to see the training code released. I am curious about the training settings e.g, learning rate, optimizer, batch size etc
